packages/frontend/src/components/Layout/NavigationSidebar.tsx

```diff
@@ -56,16 +58,63 @@ function NavigationSidebarItem({
 
 export default function NavigationSidebar() {
   const { links, closeDrawer } = useContext(LayoutNavigationContext)
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  // TODO (mal): I will make a discriminated union and combine with the drawer links if more than 1 "sidebar modal item" exists
+  const demoSidebarItem = useMemo(
+    () => (
+      <SidebarItem
+        mx={{ sm: '1rem' }}
+        w={{ lg: '16.75rem' }}
+        icon={BiBulb}
+        onClick={onOpen}
+        color="base.content.default"
+        _hover={{
+          color: 'primary.600',
+          bg: 'interaction.muted.main.hover',
+        }}
+        _active={{
+          color: 'primary.600',
+          bg: 'interaction.muted.main.active',
+        }}
+        display="flex"
+      >
+        <Text
+          textStyle="subhead-1"
+          ml={4}
+          display={{ sm: 'none', lg: 'block' }}
+        >
+          Demo
+        </Text>
+        <Badge
+          bgColor="interaction.muted.main.active"
+          color="primary.600"
+          display={{ sm: 'none', lg: 'block' }}
+        >
+          New
+        </Badge>
+      </SidebarItem>
+    ),
+    [onOpen],
+  )
 
   return (
-    <SidebarContainer>
-      {links.map((link, index) => (
-        <NavigationSidebarItem
-          key={index}
-          link={link}
-          closeDrawer={closeDrawer}
-        />
-      ))}
-    </SidebarContainer>
+    <>
+      <SidebarContainer>
+        {links.map((link, index) => (
+          <NavigationSidebarItem
+            key={index}
+            link={link}
+            closeDrawer={closeDrawer}
+          />
+        ))}
+
+        <Box pos="fixed" bottom={0}>
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/ycT2Xb5nBEjN2kf2gy95/ed4d2c81-791e-4698-a312-53fe407e130e.png)

dont use position fixed, set margin-top: auto or something

Reply:
to fix in another PR (for users who vary the screen size vertically) 😰